core/admin/mailu/configuration.py

```diff
@@ -79,6 +80,8 @@ def resolve_host(self):
         self.config['HOST_POP3'] = resolve(self.config['HOST_POP3'])
         self.config['HOST_AUTHSMTP'] = resolve(self.config['HOST_AUTHSMTP'])
         self.config['HOST_SMTP'] = resolve(self.config['HOST_SMTP'])
+        if self.config["WEBMAIL"] != "none":
+            self.config["HOST_WEBMAIL"] = resolve(self.config['HOST_WEBMAIL'])
```

Comment:
Maybe it's a bit too strict: But above this lines we use single quotes. Sorry :see_no_evil: